src/Backoffice/Employee/App/Controllers/ListEmployeesController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Routing\Controller;
+use Lightit\Backoffice\Employee\App\Resources\EmployeeResource;
+use Lightit\Backoffice\Employee\Domain\Actions\ListEmployeeAction;
+
+class ListEmployeesController extends Controller
+{
+    public function __invoke(
+        ListEmployeeAction $action,
```

Comment:
Rename the variable to the name of the class, so it is more readable (`$listEmployeeAction`)